### ISSUE TITLE
Successful Jailbreak on iPhone X 12.2 beta with 3.4.1!!

### ISSUE THREAD

**Issue by phat4dr** [pwn20wndstuff/Undecimus#1239]
Thank you very much!! I have waited for a very long time to get my device jailbroken again. You kick ass man!!! Going to go donate now!!👍

**Comment by phat4dr** [comment]
Reopening to leave more info on getting the jailbreak to work on my device. 

Device: iPhone X 
iOS: 12.2 beta
UncOver version: 3.4.1

Steps I took to jailbreak
Backup in iTunes
Erase all content and settings
Restore backup in iTunes ( note that after restore is complete, you might get a swipe up to upgrade screen, that you won’t be able to close unless you swipe up. What I did was, swipe up and right away started to do a hard reset procedure. After a couple attempts it worked.)
After restart, set up phone like normal. 
Go to iPhone storage tab in settings, and delete iOS update file.
Download and install unc0ver 3.4.1
Hit the jailbreak button wait until you are see pop up windows saying restart is needed. Hit ok
After restart
Reopen unc0ver app hit jailbreak button
Open cydia!!!

Some screen shots below. 

![A28753DC-A3DB-4E74-84F0-F11AF2199901](https://user-images.githubusercontent.com/1292013/61928052-f3c8fa00-af2b-11e9-9c4d-7be8b0e92831.png)
![D0CD48BA-7437-48B2-AE25-FF544E01F1AE](https://user-images.githubusercontent.com/1292013/61928053-f3c8fa00-af2b-11e9-963b-adaa9c2b6b1b.png)
